tests/acceptance/features/webUISharingPermissionToRoot/shareFileWithMultipleUsers.feature

```diff
@@ -11,7 +11,7 @@ Feature: Sharing files with multiple internal users with different permissions
       | Brian    |
     And user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
 
-
+  @skipOnOC10
```

Comment:
why do we skip this on oC10?

Reply:
Looks like it needs to be added to the expected failure list.
I have added it and pushed the changes